pkg/pb/metainfo.proto

```diff
@@ -35,13 +35,12 @@ service Metainfo {
 message Bucket {
     bytes                  name = 1;
     encryption.CipherSuite path_cipher = 2;
-    bytes                  attribution_id = 3;
 
```

Comment:
why did this get removed?

Reply:
we decided not to store thee attribution ID on the bucket since there is already the attribution ID table which knows about the bucket. re: https://github.com/storj/storj/blob/master/satellite/satellitedb/dbx/satellitedb.dbx#L4

Reply:
Sure, and maybe I'm mistaken, but this is the wire protocol back to clients. Clients should still be able to check the attribution id of a bucket, right?

Reply:
yes they should, but i feel like that should be a different object. If you are making a request to get the attributionID, then you only need the bucket name